tests/README.md

```diff
@@ -2,6 +2,12 @@
 
 This module contains the tests for PyDough, including unit tests, integration tests, and utility functions used in testing.
 
+## Planner Tests
+
+Many of the tests in files such as `test_qdag_conversion.py` and `test_pipeline.py` test the relational plan string generated by a PyDough query. The refsols for these tests are stored in the `test_plan_refsols` directory as text files. When these tests are run normally, the refsol is extracted from the files. However, if the environment variable `PYDOUGH_UPDATE_TESTS` is set to 1, then instead of checking against these files, the generated output is manually written to them.
+
+Make sure to set `PYDOUGH_UPDATE_TESTS` to 1 when you need to update these tests, or need to create a new one, then immediately unset it once your tests have been updated so you don't forget and think your tests are passing when they are actually failing and overriding the correct answers with nonsense.
```

Comment:
Suggestion
```suggestion
Set `PYDOUGH_UPDATE_TESTS` to 1 when updating or creating new tests. Once the tests are updated, immediately unset it to avoid unintentionally overriding correct answers with incorrect data, which may give a false impression of passing tests.
```